nep-17.mediawiki

```diff
@@ -0,0 +1,107 @@
+<pre>
+  NEP: 17
+  Title: Token Standard
+  Author: Erik Zhang <erik@neo.org>
+  Type: Standard
+  Status: Draft
+  Created: 2020-11-12
+  Replaces: 5
+</pre>
+
+==Abstract==
+
+This proposal outlines a token standard for the NEO blockchain that will provide systems with a generalized interaction mechanism for tokenized Smart Contracts.  This mechanic, along with the justification for each feature are defined.  A template and examples are also provided to enable the development community.
```

Comment:
It would be interesting to also invite previous authors of NEP-5 to join this NEP-17 for discussion, or even, authorship (if they agree too).

Reply:
P.S.: just noticed you informed that on the TODO list, requiring people to submit Pull Request. That's fine then.

Reply:
Ping @lllwvlvwlll to discuss with other authors of NEP-5.